sdk/fileapi_sdk/Common/string_utils.h

```diff
@@ -0,0 +1,42 @@
+#ifndef BASE_STRING_UTILS_H_
+#define BASE_STRING_UTILS_H_
+
+#include <memory>
+#include <string>
+
+#include "macros.h"
+
+namespace base {
+

```

Comment:
This file is copied from upe repository.
There is no point to modify those files because it will be removed

---
In reply to: [134202327](https://github.com/AzureAD/rms-sdk-for-cpp/pull/132#discussion_r134202327) [](ancestors = 134202327)